packs/windows-attacks.conf

```diff
@@ -0,0 +1,54 @@
+{
+  "platform": "windows",
+  "queries": {
+    "CCleaner_Trojan.Floxif": {
+      "query" : "select * from registry where path like 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Piriform\\Agomo%';",
+      "interval" : "3600",
+      "version": "1.4.5",
+      "description" : "(https://sensorstechforum.com/ccleaner-trojan-floxif-malware-how-to-remove/)",
+      "value" : "Artifact used by this malware"
+    },
+    "Winsecurity_info_1": {
+      "query" : "select * from programs where name = 'Winsecurity.info';",
+      "interval" : "3600",
+      "version": "1.4.5",
+      "description" : "(https://sensorstechforum.com/windows-infected-2-viruses-scam-remove/)",
+      "value" : "Artifact used by this malware"
+    },
+    "Winsecurity_info_2": {
+      "query" : "select * from chrome_extensions where name = 'Winsecurity.info';",
```

Comment:
For this and your query against `chrome_extensions` below, perhaps rewrite the query to join agains the `users` table, as otherwise you wont get back results if the query is running as `SYSTEM`, so maybe more like `select * from chrome_extensions join users using (uid) where name = 'Winsecurity.info';`